curiouslearning/FeedTheMonsterJS
# fix: CR-170 updated oromo feedback audios

# Changes
- lang/oromo/audios/amazing.mp3

# How to test
- Check oromo feedback audios.

Ref: [CR-170](https://curiouslearning.atlassian.net/jira/software/projects/CR/boards/7?selectedIssue=CR-170)


[CR-170]: https://curiouslearning.atlassian.net/browse/CR-170?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
